### ISSUE TITLE
Error after saving file PPTX file

### ISSUE THREAD

**Issue by Rita-Bubnova** [ONLYOFFICE/DocumentServer#2914]
Dear @ShockwaveNN , we have experienced the same problem multiple times with different Powerpoint presentations.

I've attached a minimal example that opens fine in Powerpoint, opens fine in ONLYOFFICE when uploaded to Nextcloud, but is defect when downloaded again via "File > Download As > PPTX". It still works fine if downloaded directly from Nextcloud.

This is with Docker image docker.io/onlyoffice/documentserver:latest, Nextcloud 29.0.7 and the ONLYOFFICE Nextcloud App version 9.4.0 (but I suspect this could also be triggered without Nextcoud at all).

[Reduced-6.pptx](https://github.com/user-attachments/files/17220462/Reduced-6.pptx)

Defect file downloaded from ONLYOFFICE: 
[Reduced-6-dl.pptx](https://github.com/user-attachments/files/17220468/Reduced-6-dl.pptx)

My initial assessment as a layperson: The slide contains an animation referencing (as far as I can tell) an object that isn't there anymore. So it is somewhat gone bad, but this was not "crafted" this way, users somehow end up in this situation by normal Powerpoint use and, more importantly, the file still works fine in Powerpoint.

_Originally posted by @adsche in https://github.com/ONLYOFFICE/DocumentServer/issues/1185#issuecomment-2387710623_

**Comment by ElenaMaaya** [comment]
@adsche, thank you for report!
I can confirm that the problem exists. I created ticket 70735  in our  issue tracker.

**Comment by ElenaMaaya** [comment]
Hello @adsche! I have a good news! This issue will be fixed in the next release ([commit](https://github.com/ONLYOFFICE/sdkjs/commit/3bce3008949b89d9a836d956d44b4aa5e5939862)). Please keep checking for updates.

**Comment by adsche** [comment]
Amazing, thank you! I'll report back when the update hits the latest docker image :)

**Comment by Rita-Bubnova** [comment]
DocumentServer v8.2.0 is released so I close this issue. Feel free to comment or reopen it if you got further questions.

**Comment by adsche** [comment]
Dear @Rita-Bubnova, many thanks for the quick bug fix!

It does work perfectly on the file I submitted (which I reduced from a previous file). 
However, on the original file, the download is still "corrupt" - but it improved slightly, as PowerPoint is at least able to repair it.

Attached a new redacted PPTX which still exhibits the problem in 8.2.0.
Same behaviour as before:
- opens fine in PowerPoint and ONLYOFFICE
- corrupt in PowerPoint after saving as PPTX from ONLYOFFICE

Will you have another look?
Thanks!

Works fine in PowerPoint:  [Reduced-8.pptx](https://github.com/user-attachments/files/17585590/Reduced-8.pptx)
Saved from OO, corrupt in PowerPoint: [Reduced-8(1).pptx](https://github.com/user-attachments/files/17585594/Reduced-8.1.pptx)
